autoscheduler/scraper/views.py

```diff
@@ -78,3 +78,21 @@ def list(self, request): # pylint: disable=arguments-differ
         return Response(formatted_data)
 
     serializer_class = CourseSearchSerializer
+
+class RetrieveGradesView(APIView):
+    """ API endpoint for viewing grade counts and avg gpa of instructor for all
+        their sections for that course. All terms taken into account
+    """
+
+    def get(self, request):
+        """ Overrides default behavior of get_object() to work using
+            instructor, subject, and course parameter in the url
+        """
+        instructor = self.request.query_params.get('instructor')
+        subject = self.request.query_params.get('subject').upper()
+        course_num = self.request.query_params.get('course_num')
+        response = Response()
+        response.data = Grades.objects.instructor_performance(
+            subject, course_num, instructor)
+
+        return response

```

Comment:
This can be changed to:
```suggestion
        data = Grades.objects.instructor_performance(subject, course_num, instructor)
        return Response(data)
```